streaming/src/main/java/org/openmrs/analytics/FhirEventWorker.java

```diff
@@ -64,6 +64,9 @@ public void process(Event event) {
 		}
 		catch (JsonParseException e) {
 			log.error(String.format("Error parsing event %s with error %s", event.toString(), e.toString()));
+		} catch (Exception e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
```

Comment:
Can you use the `log` instead? Also are you sure you want to catch any type of Exception here?

Reply:
sure